public/pages/cadastro/cadastro_pais.html

```diff
@@ -35,36 +35,36 @@
           <b>
           <ul class="nav">
             <li class="nav-item">
-              <a class="nav-link navbar-link" href="/public/pages/about.html"> QUEM SOMOS </a>
+              <a class="nav-link navbar-link" href="./public/pages/about.html"> QUEM SOMOS </a>
             </li>
             <li class="nav-item">
-              <a class="nav-link navbar-link" href="/public/pages/feed.html"> FEED </a>
+              <a class="nav-link navbar-link" href="./public/pages/feed.html"> FEED </a>
```

Comment:
Sugestão de correção:

```suggestion
              <a class="nav-link navbar-link" href="../feed.html"> FEED </a>
```